Modules/Loadable/Markups/MRML/vtkMRMLMarkupsNode.cxx

```diff
@@ -115,6 +115,7 @@ void vtkMRMLMarkupsNode::WriteXML(ostream& of, int nIndent)
 
   vtkMRMLWriteXMLBeginMacro(of);
   vtkMRMLWriteXMLBooleanMacro(locked, Locked);
+  vtkMRMLWriteXMLIntMacro(maximumNumberOfItemsInDescription, MaximumNumberOfItemsInDescription);
```

Comment:
Not neede anymore since we changed the design (as you suggested in the other comments) and the "primary" measurement is not anymore needed (i.e. I don't display anymore in the tree the Description field)